src/client/search/searchActions.js

```diff
@@ -35,9 +35,10 @@ export const RECEIVE_FEED_RESULTS = 'RECEIVE_FEED_RESULTS';
 export const searchApiForFeed = (searchTerms) =>
 (dispatch) => (
   newPromiseChain()
-  .then(() => (fetchPost('/search', searchTerms)))
-  .then(response => (response.json()))
-  .then(json => dispatch({ type: RECEIVE_FEED_RESULTS, data: json }))
+    .then(() => NProgress.start())
```

Comment:
It's magic.
Happy to remove NProgress entirely once we've put in proper semantic spinners, but it's a global object with 4 methods on it http://ricostacruz.com/nprogress/